crates/biome_js_analyze/src/lint/style/use_consistent_array_type.rs

```diff
@@ -61,10 +60,14 @@ declare_lint_rule! {
     ///     }
     /// }
     /// ```
-    /// ### Syntax
     ///
-    /// By default, all array declarations will be converted to `T[]` or `readonly T[]`, which it means `shorthand`,
-    /// or if the options is set to the "generic", that all will converted to `Array<T>` or `ReadonlyArray<T>`.
+    /// ### syntax
```

Comment:
nit: casing
```suggestion
    /// ### Syntax
```

Reply:
I get what you mean, but this is the name of the `syntax` option. As similar example, see: [`strictCase`](https://biomejs.dev/linter/rules/use-naming-convention/#strictcase) or any other option of `useNamingConvention` rule.

Reply:
Ah, you're right. Nevermind then.